mycroft/configuration/mycroft.ini

```diff
@@ -93,3 +93,10 @@ max_time = 600  # in seconds
 notify_delay = 5  # in seconds
 rate = 16000
 channels = 1
+
+[PhillipsHueSkill]
+verbose = False
```

Comment:
I'm fine with keeping it, I think the only thing would be whether it would default to false or true.